src/lib/paginator/paginator.scss

```diff
@@ -58,7 +63,15 @@ $mat-paginator-button-last-increment-icon-margin: 9px;
 
 .mat-paginator-page-size-select {
```

Comment:
@andrewseguin an alternate approach to solve this would be to lock down the form field to always have `appearance="standard"`.

Reply:
Not sure how that fits into the spec, but it seems that is valid. However, in the case of the issue it sounds like they do want the different style; I suppose we should support it